Python/pylifecycle.c

```diff
@@ -2095,12 +2092,11 @@ new_interpreter(PyThreadState **tstate_p, const PyInterpreterConfig *config)
     *tstate_p = NULL;
 
     /* Oops, it didn't work.  Undo it all. */
-    PyErr_PrintEx(0);
```

Comment:
I removed the `PyErr_PrintEx(0)` call, which never printed anything (failures are reported via PyStatus) and was also unsafe to call in some cases when the current thread did not hold the GIL.